e2e/test/commands/create/flutter_plugin/flutter_plugin_test.dart

```diff
@@ -13,7 +13,7 @@ void main() {
       final tempDirectory = Directory.systemTemp.createTempSync();
       addTearDown(() => tempDirectory.deleteSync(recursive: true));
 
-      const pluginName = 'very_good';
+      const pluginName = 'my_plugin';
```

Comment:
`dart format` no longer orders import directives, this made us change the test subject name so there are no errors when running `dart analyze`.